wherehows-web/app/utils/api/datasets/health.ts

```diff
@@ -0,0 +1,25 @@
+import { datasetUrlByUrn } from 'wherehows-web/utils/api/datasets/shared';
+import { getJSON } from 'wherehows-web/utils/api/fetcher';
+import { IHealthScoreResponse, IDatasetHealth } from 'wherehows-web/typings/api/datasets/health';
+
+/**
+ * Returns the url for dataset metadata health by urn
+ * @param urn
+ * @return {string}
+ */
+const datasetHealthUrlByUrn = (urn: string): string => `${datasetUrlByUrn(urn)}/health`;
+
+export const readDatasetHealthByUrn = async (urn: string): Promise<IDatasetHealth> => {
+  let health: IDatasetHealth;
+
+  try {
+    ({ health } = await getJSON<IHealthScoreResponse>({ url: datasetHealthUrlByUrn(urn) }));
+  } catch {
+    return {
```

Comment:
Maybe assign this object literal to `health` as an initialization value, that way you're dealing with only one variable in this function as opposed to a variable and a literal ?